kubernetes/kfserving/Huggingface_readme.md

```diff
@@ -102,6 +112,20 @@ The Explanation response is as below :
   ]
 }
 ```
+
+KFServing supports Static batching by adding new examples in the instances key of the request:
```

Comment:
Somewhere, we should add that when performing static batching this way, the user needs to make sure they're using batch size of 1 (even though more than one input instance is specified in the input). Doing otherwise will cause a failure.

Reply:
@fbbradheintz Added it